src/web/screens/teamMembers.tsx

```diff
@@ -37,18 +75,19 @@ function TeamMembers() {
   return (
     <div className='portal-team'>
       <h1>Team Members & Contacts</h1>
-      <p>View current team members below and add additional team members to access UID Portal.</p>
-      <h2>Current Team Members</h2>
+      <p className='heading-details'>
+        View current team members below and add additional team members to access Unified ID Portal.
+      </p>
+      <h2>Team Members</h2>
       <Suspense fallback={<Loading />}>
         <Await resolve={data.users}>
```

Comment:
I found we are still returning all users, I understand with `defer, useLoaderData`, it is hard to retrieve users in the participant, should we change this? Or there is another ticket for this?

Reply:
I've left it this way for now for convenience so the test users can see each other - I'm planning to do a new ticket to change this and we'll set up some better test data in integ as well. I've added UID-1357 to the backlog to address this.